semanticcpg/src/main/scala/io/shiftleft/semanticcpg/passes/cfgdominator/CfgDominatorPass.scala

```diff
@@ -2,10 +2,9 @@ package io.shiftleft.semanticcpg.passes.cfgdominator
 
 import io.shiftleft.codepropertygraph.Cpg
 import io.shiftleft.codepropertygraph.generated.EdgeTypes
-import io.shiftleft.codepropertygraph.generated.nodes.{Method, StoredNode}
+import io.shiftleft.codepropertygraph.generated.nodes._
```

Comment:
in general it's preferrable to avoid wildcard imports, to avoid surprises and improve readability
there are some exceptions, most importantly when importing implicits of course

Reply:
No. I will not manually import every class used, especially since CI throws errors on unused imports or imports that are not alphabetically ordered or such bullshit.

This is a hill I am willing to die on.

You can have `import io.shiftleft.codepropertygraph.generated.nodes._` everywhere, or you can have ` import io.shiftleft.codepropertygraph.generated.nodes` and `nodes.Method`; choose your poison.

Reply:
Re #1233 : 

I wholeheartedly agree that renaming the extension classes is a good idea. I don't mind if you write code with explicit imports of specific classes, and then refer to node types with unqualified name.

I strongly reject to making this a mandatory style for all our code, and this was not proposed in https://github.com/ShiftLeftSecurity/codepropertygraph/issues/1233 -- your issue proposed making unqualified names acceptable, not making qualified names unacceptable.

Reply:
Ok, per slack. not dying on this hill anymore, will fix.